Develop

nedemenang opened the pull request.

Commit:
Merge branch 'develop' of https://github.com/nedemenang/post-it-app into develop

Commit:
update package.json

nedemenang committed:
Merge pull request #10 from nedemenang/codereview_corrections

update package.json

Commit:
npm start working

nedemenang committed:
Merge pull request #11 from nedemenang/codereview_corrections

npm start working

Commit:
finish set up

nedemenang committed:
Merge pull request #12 from nedemenang/codereview_corrections

finish set up

Commit:
Merge branch 'develop' of https://github.com/nedemenang/post-it-app into develop

Commit:
Edit read me

Commit:
troubleshoot hound

nedemenang committed:
Merge pull request #13 from nedemenang/codereview_corrections

Edit read me

Commit:
troubleshoot hound

Commit:
troubleshoot hound

Commit:
trouble shooting hound

nedemenang committed:
Merge pull request #14 from nedemenang/codereview_corrections

Troubleshoot Hound

Commit:
trouble shoot hound 2

nedemenang committed:
Merge pull request #15 from nedemenang/codereview_corrections

trouble shoot hound 2

Commit:
Corrected server.js

nedemenang committed:
Merge pull request #16 from nedemenang/codereview_corrections

Corrected server.js

Commit:
Read me file

nedemenang committed:
Merge pull request #17 from nedemenang/codereview_corrections

Read me file

Commit:
Created sign out route and added rsult status

nedemenang committed:
Merge pull request #18 from nedemenang/signOut

Sign out

Commit:
write test and build

nedemenang committed:
Merge pull request #19 from nedemenang/test

write test and build

Commit:
adding test for more functionality

nedemenang committed:
Merge pull request #20 from nedemenang/test2

adding test for more functionality

Commit:
Post message route

nedemenang committed:
Merge pull request #21 from nedemenang/login_component

Post message route

Commit:
convert to ES6

Commit:
Create front end components, correct some errors

nedemenang committed:
Merge pull request #22 from nedemenang/login_component

Login component

Commit:
Add State managment, create fetch data routes

nedemenang committed:
Merge pull request #23 from nedemenang/state_management

Add State managment, create fetch data routes

Commit:
refactor components using material ui

nedemenang committed:
Merge pull request #24 from nedemenang/refactor_components_materialui

refactor components using material ui

Commit:
more state management and receiving data

Commit:
login, sign up and add group component working

Commit:
Returned logging success

Commit:
fix register login success label

Commit:
transpile to dist folder

nedemenang committed:
Merge pull request #25 from nedemenang/data_retrieval

Data retrieval

Commit:
build distribution folder

nedemenang committed:
Merge pull request #26 from nedemenang/data_retrieval

build distribution folder

Commit:
bundle .js file

nedemenang committed:
Merge pull request #27 from nedemenang/data_retrieval

bundle .js file

Commit:
package.json deploy

nedemenang committed:
Merge pull request #28 from nedemenang/data_retrieval

package.json deploy

Commit:
deploy for heroku

nedemenang committed:
Merge pull request #29 from nedemenang/data_retrieval

deploy for heroku

Commit:
deploy for heroku 2

nedemenang committed:
Merge pull request #30 from nedemenang/data_retrieval

deploy for heroku 2

Commit:
deploy for heroku 3

nedemenang committed:
Merge pull request #31 from nedemenang/data_retrieval

deploy for heroku 3

Commit:
deploy for heroku 4

nedemenang committed:
Merge pull request #32 from nedemenang/data_retrieval

deploy for heroku 4

Commit:
deploy for heroku 5

nedemenang committed:
Merge pull request #33 from nedemenang/data_retrieval

deploy for heroku 5

Commit:
deploy to heroku 6

nedemenang committed:
Merge pull request #34 from nedemenang/data_retrieval

deploy to heroku 6

Commit:
deploy to heroku 7

nedemenang committed:
Merge pull request #35 from nedemenang/data_retrieval

deploy to heroku 7

Commit:
deploy for heroku 8

nedemenang committed:
Merge pull request #36 from nedemenang/data_retrieval

deploy for heroku 8

Commit:
deploy for heroku 9

nedemenang committed:
Merge pull request #37 from nedemenang/data_retrieval

deploy for heroku 9

Commit:
get group messages

Commit:
deploy to heroku 10

Commit:
readme

nedemenang committed:
Merge pull request #38 from nedemenang/data_retrieval

Data retrieval

Commit:
deploy to heroku

nedemenang committed:
Merge pull request #39 from nedemenang/data_retrieval

deploy to heroku

Commit:
deploy to heroku

nedemenang committed:
Merge pull request #40 from nedemenang/data_retrieval

deploy to heroku

Commit:
deploy to heroku

nedemenang committed:
Merge pull request #41 from nedemenang/data_retrieval

deploy to heroku

Commit:
final deploy to heroku for today

nedemenang committed:
Merge pull request #42 from nedemenang/data_retrieval

final deploy to heroku for today

Commit:
message functionality working

nedemenang committed:
Merge pull request #43 from nedemenang/data_retrieval

message functionality working

Commit:
deploy to heroku

nedemenang committed:
Merge pull request #44 from nedemenang/data_retrieval

deploy to heroku

Commit:
Deploy to heroku 2

nedemenang committed:
Merge pull request #45 from nedemenang/data_retrieval

Deploy to heroku 2

Commit:
deploy to heroku 3

nedemenang committed:
Merge pull request #46 from nedemenang/data_retrieval

deploy to heroku 3

Commit:
error handling

Commit:
sign out feature

nedemenang committed:
Merge pull request #47 from nedemenang/data_retrieval

Data retrieval

Commit:
list users not in group functionality

nedemenang committed:
Merge pull request #48 from nedemenang/data_retrieval

list users not in group functionality

Commit:
get users not in group functionality

Commit:
add new message to user profile

nedemenang committed:
Merge pull request #49 from nedemenang/data_retrieval

Data retrieval

Commit:
local storage installation

Commit:
persist user object using local storage

nedemenang committed:
Merge pull request #50 from nedemenang/data_retrieval

Data retrieval

Commit:
finish google registration

nedemenang committed:
Merge pull request #51 from nedemenang/component_test

finish google registration

Commit:
password reset component

Commit:
real time functionality

Commit:
Is Read functionality

Commit:
get user read messages, other fixes

nedemenang committed:
Merge pull request #52 from nedemenang/component_test

Component test

Commit:
Real time partialy working

Commit:
Improve realtime

nedemenang committed:
Merge pull request #53 from nedemenang/improve_realtime3

Improve realtime3

Commit:
Fix Email

nedemenang committed:
Merge pull request #54 from nedemenang/fix_email

Fix Email

Commit:
Fix navigation bar

nedemenang committed:
Merge pull request #55 from nedemenang/write_tests

Fix navigation bar

Commit:
Some ux changes

nedemenang committed:
Merge pull request #56 from nedemenang/write_tests

Some ux changes

Commit:
test and notification

Commit:
added notification changes

nedemenang committed:
Merge pull request #57 from nedemenang/write_tests

Write tests

Commit:
travis file

Commit:
updated snapshot

Commit:
write jsdocs, include test setup

Commit:
More Test

nedemenang committed:
Merge pull request #58 from nedemenang/write_tests

Write tests

Commit:
Fix environment variables

Commit:
babel register

nedemenang committed:
Merge pull request #59 from nedemenang/write_tests

Fix environment variables

Commit:
Fixing travis ci build

Commit:
Travis ci build

Commit:
travis badge

nedemenang committed:
Merge pull request #60 from nedemenang/write_tests

Write tests

Commit:
coverall badge

Commit:
Adjust coverall badge

nedemenang committed:
Merge pull request #61 from nedemenang/write_tests

Write tests

Commit:
code climate

Commit:
fix promise error

Commit:
promise fix

Commit:
fix travis ci build

Commit:
fix promise error

Commit:
fix travis ci

nedemenang committed:
Merge pull request #62 from nedemenang/write_tests

Write tests

Commit:
Travis to work

nedemenang committed:
Merge pull request #63 from nedemenang/write_tests

Travis to work

Commit:
configure nyc

Commit:
update travis file

Commit:
Write component js docs

Commit:
Test coverage

nedemenang committed:
Merge pull request #64 from nedemenang/write_tests

Write tests

emmyakin commented on Client/components/App.jsx:
```diff
@@ -0,0 +1,119 @@
+  /**
+  * @return {void} return void
+  */
+componentUnmount() {
```
componentWillUnmount

emmyakin commented on Client/components/App.jsx:
```diff
@@ -0,0 +1,119 @@
+ * @param {object} props props objects
+  * @return {void} return void
+  */
+  constructor(props){
```
this should come before componentDiMount and ComponentWillUnmount

emmyakin commented on Client/components/App.jsx:
```diff
@@ -0,0 +1,119 @@
+/**
+  * @return {function} getappstate function
+  */
+  getInitialState(){
```
you do not need getInitialState if you are using class component, this happens in the constructor as
this.state = getAppState()

emmyakin commented on Client/components/App.jsx:
```diff
@@ -0,0 +1,119 @@
+  /**
+  * @return {JSX} App page
+  */
+   render() {
```
render function should be the last method in a component
i think you should add "eslint-plugin-react" to your eslint file

emmyakin commented on Client/components/App.jsx:
```diff
@@ -0,0 +1,119 @@
+
+injectTapEventPlugin();
+
+const muiTheme = getMuiTheme({
```
unused const

emmyakin commented on Client/actions/AppActions.js:
```diff
@@ -0,0 +1,168 @@
+import AppDispatcher from '../dispatcher/AppDispatcher';
+import AppConstants from '../constants/AppConstants';
```
you can destructure the constants here as 
`import { LOGIN_USER, .... } from '../constants/AppConstants';`
so you would not need to write AppConstants.LOGIN_USER ..

emmyakin commented on Client/components/Group.jsx:
```diff
@@ -0,0 +1,91 @@
+    AppAPI.getQuickGroupMessages(userGroup);
+    AppAPI.getUsersNotInGroups(this.props.group);
+
+    const updateObject = {
```
unused const

emmyakin commented on Client/components/Group.test.js:
```diff
@@ -0,0 +1,53 @@
+
+
+describe("the group item component", () => {
+        let mountedComponent;
```
indentations looks off in this file

emmyakin commented on Client/components/Group.test.js:
```diff
@@ -0,0 +1,53 @@
+            profilePic: "//localhost:3000/static/files/blank-profile-pic.png" 
+        };
+
+        const groupItem = () => {
```
rather than this, just use this line in beforeEach, since it will be called before every test
`mountedComponent = shallow(<Group group = {group} loggedInUser = {loggedInUser} key = {1}/>);`

emmyakin commented on Client/components/Group.test.js:
```diff
@@ -0,0 +1,53 @@
+it('should match snapshot test', function() {
+  const component = groupItem();
+  const tree = toJson(component);
+  expect(tree).toMatchSnapshot();
```
not sure if this is a use case for toMatchSnapshot

this test probably needs to check that component contains <Icon> when newMessage is true

emmyakin commented on Client/components/Group.jsx:
```diff
@@ -0,0 +1,91 @@
+
+
+
+class Group extends Component {
```
this component does not use any state, it might as well be written as a functional component

emmyakin commented on Client/actions/AppActions.js:
```diff
@@ -0,0 +1,168 @@
+
+const AppActions = {
+ // console.log('Logging user....');
+  login(user) {
```
consider using named exports rather than a single default exports in this file
e.g export const logIn = (user) = > {......}
why? with named exports you can easily destructure imports, in order to import only methods/functions you need in files

emmyakin commented on Client/components/App.jsx:
```diff
@@ -0,0 +1,119 @@
+import React, {Component} from 'react';
+import ReactDOM from 'react-dom';
+import Login from './Login';
+import AppActions from '../actions/AppActions';
```
unused import

emmyakin commented on Client/components/Group.jsx:
```diff
@@ -0,0 +1,91 @@
+  */
+  render(){
+    let newM = '';
+    if(this.props.group.newMessage === true){
```
no need to compare values in if statements with boolean
why? if(x) , in this statement, x is evaluated as Boolean(x) value by the if construct
so, do if(x) rather than if(x === true)

emmyakin commented on Client/components/Group.jsx:
```diff
@@ -0,0 +1,91 @@
+} from 'react-router-dom';
+import '../public/style.css';
+import $ from '../public/jquery.js';
+import AppActions from '../actions/AppActions';
```
named exports above in '../actions/AppActions' will allow for this
import { selectGroup } from '../actions/AppActions'

emmyakin commented on Client/components/Group.jsx:
```diff
@@ -0,0 +1,91 @@
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import {green100, green500, green700} from 'material-ui/styles/colors';
+import AppStore from '../stores/AppStore';
+import AppAPI from '../utils/appAPI';
```
same for API, named exports and de-structured imports

emmyakin commented on Client/components/GroupForm.jsx:
```diff
@@ -0,0 +1,78 @@
+
+
+
+class GroupForm extends Component {
```
this component does not use state, try functional component here.

emmyakin commented on Client/components/GroupForm.jsx:
```diff
@@ -0,0 +1,78 @@
+      mm = '0'+mm
+  } 
+
+  today = mm + '/' + dd + '/' + yyyy;
```
rather than line 25-37, you can use
const today = (new Date()).toLocaleString('en-GB')

emmyakin commented on Client/components/GroupForm.jsx:
```diff
@@ -0,0 +1,78 @@
+
+  const user = localStorage.getItem('user');
+
+ if(this.refs.group.value !== '')
```
this will fail If I add empty spaces in the input

emmyakin commented on Client/components/GroupForm.jsx:
```diff
@@ -0,0 +1,78 @@
+
+  today = mm + '/' + dd + '/' + yyyy;
+
+  const groupname = this.refs.group.value.trim();
```
rather than refs, why not use state, or gran the value from event

emmyakin commented on Client/components/Group.jsx:
```diff
@@ -0,0 +1,91 @@
+  */
+groupClicked() {
+ 
+    const user = localStorage.getItem('user');
```
why not keep user in store too

emmyakin commented on Client/components/GroupForm.jsx:
```diff
@@ -0,0 +1,78 @@
+  const groupname = this.refs.group.value.trim();
+  const dateCreated = today;
+
+  const user = localStorage.getItem('user');
```
better to use JSON.parse(localStorage.getItem('user')), so you don't have to repeat JSON.parse below

emmyakin commented on Client/components/Login.jsx:
```diff
@@ -0,0 +1,196 @@
+    profilePic: ''
+  }
+
+  if(this.refs.loginEmail.value === ''){
```
use state and onChange handlers rather than refs

emmyakin commented on Client/components/Login.jsx:
```diff
@@ -0,0 +1,196 @@
+   * @memberof Login
+   */
+  componentDidMount() {
+    window.addEventListener('google-loaded',this.renderGoogleLoginButton);
```
add a componentWillUnmount to removeEventListener

emmyakin commented on Client/components/Login.jsx:
```diff
@@ -0,0 +1,196 @@
+   * @param {object} props property object
+   * @memberof Login
+   */
+  constructor(props){
```
constructor should always come first

emmyakin commented on Client/components/Login.jsx:
```diff
@@ -0,0 +1,196 @@
+   * @returns {JSX} return login page
+   * @memberof Login
+   */
+  render(){
```
hmm indentation

emmyakin commented on Client/components/LoginMessageBoard.jsx:
```diff
@@ -0,0 +1,109 @@
+   * @returns {void} return void
+   * @memberof LoginMessageBoard
+   */
+  getInitialState(){
```
delete

emmyakin commented on Client/components/LoginMessageBoard.jsx:
```diff
@@ -0,0 +1,109 @@
+   * @return {void} return void
+   * @memberof LoginMessageBoard
+   */
+  componentUnmount() {
```
componentWillUnmount

emmyakin commented on Client/components/LoginMessageBoard.jsx:
```diff
@@ -0,0 +1,109 @@
+    */
+   render() {
+    
+    if (this.state.isAuthenticated == true)
```
see prev comment with if

emmyakin commented on Client/components/LoginMessageBoard.jsx:
```diff
@@ -0,0 +1,109 @@
+    }
+     
+    let componentToMount; 
+    if(localStorage.getItem('user') == null) {
```
I would rather advise you separate into separate route and and use an authWrapper to redirect to Login Page or MessageBoard

emmyakin commented on Client/components/User.jsx:
```diff
@@ -0,0 +1,90 @@
+   */
+  render(){
+    return(
+      <MuiThemeProvider muiTheme={muiTheme}>
```
MuiThemeProvider should be at the root of your app to avoid repetition in every component

emmyakin commented on Client/components/notification.jsx:
```diff
@@ -0,0 +1,109 @@
+   */
+  componentDidMount(){
+
+      this.socket = io('http://localhost:3000');
```
need to abstract this into the actions or store

emmyakin commented on Client/components/passwordReset.jsx:
```diff
@@ -0,0 +1,60 @@
+  // 
+  event.preventDefault();
+  const email = this.refs.resetEmail.value.trim();
+  if(this.refs.resetEmail.value === ''){
```
avoid using refs

emmyakin commented on Client/components/sampleTest.js:
```diff
@@ -0,0 +1,66 @@
+describe('GroupListView', () => {
+  let mountedComponent;
+  let props;
+  const groupListView = () => {
```
refer to comment above

emmyakin commented on Client/index.jsx:
```diff
@@ -0,0 +1,9 @@
+import App from './components/App';
+import Signup from './components/Signup';
+
+ReactDOM.render(<App />, document.getElementById('app'));
```
which is which Client/index.jsx or Client/main.js

emmyakin commented on Client/utils/appAPI.js:
```diff
@@ -0,0 +1,273 @@
+      };
+      AppActions.receiveSuccess(response.data.message);
+      AppActions.receiveAuthenticatedUser(authuser);
+      AppActions.receiveErrors('');
```
no need to call this line

emmyakin commented on Client/utils/appAPI.js:
```diff
@@ -0,0 +1,273 @@
+  registerNewUser(user) {
+    console.log(user);
+    axios.post('/users/signup', {
+      email: user.email,
```
just pass user

emmyakin commented on Client/utils/appAPI.js:
```diff
@@ -0,0 +1,273 @@
+
+  updateUserProfile(user) {
+    // console.log(user);
+    axios.post('/users/updateUserProfile', {
```
axios.post('/users/updateUserProfile', user).then(....

emmyakin commented on Client/utils/appAPI.js:
```diff
@@ -0,0 +1,273 @@
+import AppActions from '../actions/AppActions';
+
+module.exports = {
+  registerNewUser(user) {
```
use named exports here and below...

emmyakin commented on Client/utils/appAPI.js:
```diff
@@ -0,0 +1,273 @@
+  createNewGroup(group) {
+    console.log(group);
+    axios.post('/group', {
+      groupName: group.groupname,
```
pass group and ensure cases match e.g groupName and group.groupName

emmyakin commented on Client/utils/appAPI.js:
```diff
@@ -0,0 +1,273 @@
+  getGroupMessages(userGroup) {
+    // console.log(`user/${userGroup.userId}/group/${userGroup.groupId}/messages`);
+    console.log('get group messages');
+    axios.get(`/user/${userGroup.userId}/group/${userGroup.groupId}/messages`);
```
incomplete

emmyakin commented on Server/Controllers/groups.js:
```diff
@@ -0,0 +1,341 @@
+   * @returns {Response} response object
+   */
+  addUser(req, res, firebase) {
+    const userlogin = firebase.auth().currentUser;
```
is it possible that the currentUser is not the same user making the request

Commit:
Code review corrections

Commit:
Change to staging variables

Commit:
staging app

Commit:
set up environment variables

Commit:
environment variables

emmyakin commented on postittest-45ed5-firebase-adminsdk-7fjno-42a736db9f.json:
```diff
@@ -0,0 +1,12 @@
+{
```
this file should not be committed

emmyakin commented on Server/server.js:
```diff
@@ -0,0 +1,68 @@
+require('dotenv').config();
+// dotenv.load();
+
+require('es6-promise').polyfill();
```
do you need this?

emmyakin commented on package.json:
```diff
@@ -0,0 +1,120 @@
+  "main": "index.js",
+  "scripts": {
+    "dev": "babel-watch server.js",
+    "start": "NODE_ENV=development webpack && babel-node ./Server/server.js",
```
you do not need to run webpack in this script command

nedemenang committed:
Merge branch 'staging' into code_review

nedemenang committed:
Merge pull request #68 from nedemenang/code_review

Code review

emmyakin reviewed (commented):
Remove all console.logs, gitignore the dist and coverage folder

Commit:
fix error

Commit:
implementing feedback 8sep

Commit:
Correct sms key error

Commit:
Code review corrections

Commit:
bug fix(google login): Users should be able to log in via google +

Change componentwillmount to componentwillunmount

Commit:
bug-fix/login

Use state management for form fields
Format js docs
add some tests

Commit:
Bug Fix(Implement defense feedback)

Add codeclimate.yml file

Commit:
bug-fix(Implement defense feedback):

Add code climate badge in readme

Commit:
bug-fix(Implement defense feedback)

Add coverage command to jest tests

Commit:
bug-fix/implement-defense-feedback

Write Component Tests for various components

Commit:
bug-fix/implement-defense-feedback

Snapshots for component Tests

Commit:
bug-fix/implement-defense-feedback

Implement Server test

Commit:
bug-fix/implement-defense-feedback

Use codcov for coverage reports

Commit:
bug-fix/implement-defense-feedback

Edit JsDocs to Airbnb standard

Commit:
bug-fix/implement-defense-feedback

Coverage roports

Commit:
bug-fix/implement-defense-feedback

Remove coverall badge

Commit:
bug-fix/implement-defense-feedback

Fix travis ci bug

Commit:
bug-fix/implement-defense-feedback

upload coverage report

nedemenang committed:
Merge pull request #70 from nedemenang/bug-fix/implement-defense-feedback

Bug fix/implement defense feedback

Commit:
bug-fix/implement-defense-feedback

Add codecove badge

Commit:
bug-fix/implement-defense-feedback

Add codecov badge

nedemenang committed:
Merge pull request #71 from nedemenang/bug-fix/implement-defense-feedback

Bug fix/implement defense feedback

nedemenang committed:
Add a license file

Commit:
bug-fix/implement-defense-feedback

Edit markdowns in readme

Commit:
bug-fix/implement-defense-feedback

Add MIT license file

Commit:
Add MIT license file

Commit:
bug-fix/implement-defense-feedback

Add markdown in readme file

Commit:
bug-fix/implement-defense-feedback

Remove confirmation box from user
Add material UI confirmation box adding user

Commit:
bug-fix/implement-defense-feedback

Fix console errors

Add toastr for notification

Commit:
bug-fix/implement-defense-feedback

fix profile picture file space

nedemenang committed:
Merge pull request #72 from nedemenang/bug-fix/implement-defense-feedback

Bug fix/implement defense feedback

Commit:
bug-fix/implement-defense-feedback

Add production webpack

Change send message button style

Commit:
bug-fix/implement-defense-feedback

add toastr for better error handling

update snapshots for failing component test

Commit:
bug-fix/implement-defense-feedback

Add test for password reset

Commit:
bug-fix/implement-defense-feedback

Add more test for group

nedemenang committed:
Merge pull request #74 from nedemenang/bug-fix/implement-defense-feedback

Bug fix/implement defense feedback

Commit:
bug-fix/implement-defense-feedback

make UI mobile responsive

nedemenang committed:
Merge pull request #75 from nedemenang/bug-fix/implement-defense-feedback

bug-fix/implement-defense-feedback

Commit:
bug-fix/implement-defense-feedback

Convert css to scss

Commit:
bug-fix/implement-defense-feedback

coverage for test

nedemenang committed:
Merge pull request #76 from nedemenang/bug-fix/implement-defense-feedback

Bug fix/implement defense feedback

Commit:
bug-fix/implement-defense-feedback

Fix login image error

nedemenang committed:
Merge pull request #77 from nedemenang/bug-fix/implement-defense-feedback

bug-fix/implement-defense-feedback

Commit:
bug-fix/implement-defense-feedback

Update coverage information

Commit:
bug-fix/implement-defense-feedback

Separate firebase config file from index route file

Commit:
bug-fix/implement-defense-feedback

Use pop up to sign in with google

Commit:
bug-fix/implement-defense-feedback

Fix production webpack issue

Commit:
bug-fix/implement-defense-feedback

Fix google signin on heroku

Commit:
bug-fix/implement-defense-feedback

Fix dotenv webpack issue

Commit:
bug-fix/implement-defense-feedback

Fix production api key bug

Commit:
bug-fix/implement-defense-feedback

add node_env=production in start script

Commit:
bug-fix/implement-defense-feedback

add console.log to view api key

Commit:
bug-fix/implement-defense-feedback

Fix heroku webpack issue

Commit:
bug-fix/implement-defense-feedback

fix webpack error issue

Commit:
bug-fix/implement-defense-feedback

Fix heroku time out issue

Commit:
bug-fix/implement-defense-feedback

Fix heroku webpack issue

Commit:
bug-fix/implement-defense-feedback

Fix heroku deploy error

Commit:
bug-fix/implement-defense-feedback

Add dotenv loading to webpack config

nedemenang committed:
Merge pull request #78 from nedemenang/bug-fix/implement-defense-feedback

Bug fix/implement defense feedback

Commit:
bug-fix/Implement-second-defense-feedback

Change folder structure names
Write more comprehensive test
Adjust read me file
Implement feed back from defense

Commit:
bug-fix/Implement-second-defense-feedback

Make application mobile reponsive

Commit:
bug-fix/Implement-second-defense-feedback

Add MIT license

Commit:
bug-fix/Implement-second-defense-feedback

Point coverage badge to correct branch

Commit:
bug-fix/Implement-second-defense-feedback

Edit read me file

Commit:
bug-fix/Implement-second-defense-feedback

Change case of client and server folder

Commit:
bug-fix/Implement-second-defense-feedback

adding coverage files

Commit:
bug-fix/fix-directory-naming

point read me to correct branch

Commit:
bug-fix/fix-directory-naming

Remaning client and server folders

Commit:
bug-fix/fix-directory-naming

Rename folders

Commit:
bug-fix/fix-directory-naming

Change names of notificaiton and passwordreset files

Commit:
bug-fix/fix-directory-naming

Change file path for server utility folder

Commit:
bug-fix/fix-directory-naming

change folder names to correct casing

Commit:
bug-fix/fix-directory-naming

Mocking image files

Commit:
bug-fix/fix-directory-naming

Mock image for test

Commit:
bug-fix/fix-directory-naming

fix mocking error

Commit:
bug-fix/fix-directory-naming

fix image mockery issue

Commit:
bug-fix/fix-directory-naming

add clean cache to mockery parameters

Commit:
bug-fix/fix-directory-naming

add mockery to before hook

Commit:
bug-fix/fix-directory-naming

add mock to login test file

Commit:
bug-fix/fix-directory-naming

fix image upload issue

Commit:
bug-fix/fix-directory-naming

remove login image

Commit:
bug-fix/fix-directory-naming

point readme badge to correct branch

Commit:
bug-fix/fix-directory-naming

fix undefined group in notification

Commit:
bug-fix/fix-directory-naming

add image url to login image

Commit:
bug-fix/fix-directory-naming

Fix sign in error

Commit:
bug-fix/fix-directory-naming

add brower history for redirecting

Commit:
bug-fix/fix-directory-naming

fix failing tests

Commit:
bug-fix/fix-directory-naming

Fix update user profile error

Commit:
bug-fix/fix-directory-naming

fix some validation errors

Commit:
bug-fix/fix-directory-naming

adding jsdocs to some components

Commit:
bug-fix/fix-directory-naming

New testing for props

Commit:
bug-fix/fix-directory-naming

Fix maximum character exceeded in test

Commit:
bug-fix/fix-directory-naming

implement middleware
fix adding user to group

Commit:
bug-fix/fix-directory-naming

add middleware to other routes

Commit:
bug-fix/fix-directory-naming

Fix password reset test

Commit:
bug-fix/fix-directory-naming

implement all feedback given

Commit:
bug-fix/fix-directory-naming

fix group controller reference

nedemenang committed:
Merge pull request #80 from nedemenang/bug-fix/Fix-directory-naming

Bug fix/fix directory naming
